.env.example

```
@@ -1,3 +1,7 @@
-# No environment variables needed.
-# API keys are provided by users at runtime in the browser.
-# Keys are stored only in React state / sessionStorage — never sent to any backend.
+# Provider API keys are entered by users at runtime in the browser.
+# They are stored only in React state / sessionStorage unless a scheduled job is created.
+
+# Required for Supabase-backed workflow features.
+# Core agent runs work without these values.
+VITE_SUPABASE_URL=your_supabase_url
+VITE_SUPABASE_ANON_KEY=your_supabase_anon_key
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_SUPABASE_ANON_KEY key should go before the VITE_SUPABASE_URL key

(UnorderedKey)